src/core/AgnLocusStream.c

```diff
@@ -261,7 +261,8 @@ static void locus_stream_extend(AgnLocusStream *stream, AgnLocus *locus)
     {
       agn_locus_set_range(locus, locusrange.start - stream->delta,
                           locusrange.end);
-      if(stream->endmode >= 0 && !stream->skip_iiLoci)
+      if(stream->endmode >= 0 && !stream->skip_iiLoci && 
+				locusrange.start - stream->delta - 1 > 0)
```

Comment:
Looks like many of the changes to this file are cosmetic (line width). This is one of the few changes related to length accounting, right?